src/AutoRest.CSharp/Common/Generation/Types/TypeFactory.cs

```diff
@@ -177,7 +179,9 @@ internal static bool IsOperationOfAsyncPageable(CSharpType type)
         internal static bool IsOperationOfPageable(CSharpType type)
             => type.IsFrameworkType && type.FrameworkType == typeof(Operation<>) && type.Arguments.Length == 1 && IsPageable(type.Arguments[0]);
 
-        internal static Type? ToFrameworkType(Schema schema) => schema.Type switch
+        internal static Type? ToFrameworkType(Schema schema) => ToFrameworkType(schema, schema.Extensions);
+
+        internal static Type? ToFrameworkType(Schema schema, RecordOfStringAndAny? schemaExtensions) => schema.Type switch
```

Comment:
Same comment here, but since this is an internal method, we do not need to add an xml document, but it is always a good thing to add comments describing things